src/modules/ROOT/pages/changelog/202309/07.json

```diff
@@ -0,0 +1,141 @@
+{
+  "meta": {
+    "releasedOn": "2024.06.14"
+  },
+
+  "added": [
+    {
+      "what": [
+        "the option to stack values of multiple number fields in {APP_NAME_REPORTER} chart."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1780"
+      ]
+    },
+    {
+      "what": [
+        "the option to clone a report in {APP_NAME_REPORTER}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1748"
+      ]
+    },
+    {
+      "what": [
+        "the option to provide an `id` attribute to a page block in {APP_NAME_COMPOSE}."
+      ],
+      "why": [
+        "to provide the user with the option to select and customize a page block."
+      ],
```

Comment:
Not sure this makes sense to the reader; could you elaborate on what exactly is going on here? We can already customize page blocks so what does this ID change?